azure-iot-device/azure/iot/device/iothub/abstract_clients.py

```diff
@@ -67,8 +68,13 @@ def create_from_connection_string(cls, connection_string, ca_cert=None, **kwargs
         authentication_provider = auth.SymmetricKeyAuthenticationProvider.parse(connection_string)
         authentication_provider.ca_cert = ca_cert  # TODO: make this part of the instantiation
         pipeline_configuration = IoTHubPipelineConfig(**kwargs)
+        upload_pipeline = None
+        if pipeline_configuration.blob_upload:
```

Comment:
>if pipeline_configuration.blob_upload: [](start = 8, length = 38)

Do we really need to give the user the ability to decide that they don't want the blob upload client or not?  I know we do this in other languages, and I think we do it to keep packages small.  I don't think we have the same need in Python.